0.preprocess-sites/README.md

```diff
@@ -30,5 +30,5 @@ This step compiles all the results from the spot and cell processing and outputs
 ## Step 4 - Image and Segmentation Quality Control
 
 This step outputs a series of QC metrics pertaining to the actual image acquisition and image segmentation quality.
-The QC metrics are visualized by site across wells per experiment. 
+The QC metrics are visualized by site across wells per experiment.
 It also outputs lists of sites that trigger QC flags for confluence, image saturation, and image alignment.
```

Comment:
Is your intent for this README to have each sentence on its own line (as we had discussed for the site_processing_config_README in the template)? If so, .md requires two spaces at the end of each line to make a line break.

Reply:
oh! Sorry - "one-sentence-per-line refers" to markdown style exclusively. My intent is for them to appear in paragraph form in the rendered markdown, but that in the raw markdown each sentence belongs on its own line.